css/class-naming-conventions/readme.md

```diff
@@ -43,23 +44,23 @@ Our convention (which we call CSM or Component, Sub-Component, Modifier) uses [B
 
 This example may seem confusing at first but if we break down each of the selectors that we have, it begins to make more sense.
 
-`.c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
+`c-blog` This is a component. It describes a high level module or component. In this instance, it describes the container for all of our blog posts.
 
-`.c-blog__title` This is a sub-component. It's always a child of a module or component. In this instance, it is a title for our blog post container
+`c-blog__title` This is a sub-component. It's always a child of a module or component. In this instance, it is a title for our blog post container
 
-`.c-blog-post` This is another component. This one describes a specific blog post. We make this its own component because a blog post is not necessarily a child of the blog container. It can and should be able to live independently.
+`c-blog-post` This is another component. This one describes a specific blog post. We make this its own component because a blog post is not necessarily a child of the blog container. It can and should be able to live independently.
 
-`.c--featured` This is a modifier. It is always chained to a component or sub-component. In this instance, it describes a different way of displaying a component.
+`c--featured` This is a modifier. It is always chained to a component or sub-component. In this instance, it describes a different way of displaying the `c-blog-post` component.
 
-`.c-blog-post__time` Like before, this is another sub-component. This time it belongs to the c-blog-post. It's still a subcomponent even though it is not a direct child of the component.
+`c-blog-post__time` Like before, this is another sub-component. This time it belongs to the c-blog-post. It's still a sub-component even though it is not a direct child of the component.
 
 ### Components
 
-The highest level of a module — it should describe an independent module that you are creating. Components should be able to exist on their own or within other components. They should always live at the root level of a file.
+The highest level of a module — it should describe an independent, self-contained module. Components should be able to exist on their own or within other components. They are only responsible for itself or for what is within itself, never for anything external to itself. Component classes live at the root level of a file.
```

Comment:
Components are independent and self-contained. Styles belonging to a component should only affect the component itself, and any of its sub-components. They shouldn't affect anything external to them, or any other components that might be nested within them. (?)

Or change 'itself' to 'themselves'. :)